src/main/java/net/mcreator/element/types/Item.java

```diff
@@ -75,6 +75,7 @@
 
 	public boolean enableMeleeDamage;
 	public double damageVsEntity;
+	public double attackSpeed;

```

Comment:
No need to converter, I think it is safe to just set default value to 1.6 in constructor, as this value has no impact of melee damage is enabled anyways

Reply:
But if I don't make a converter, the constructor would not set the field's value to 1.6 for existing elements. Then if someone opened the item, the attack speed would be 0.